File: DiplomskiAPI/Models/WorkOrderPartItem.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DiplomskiAPI.Models
{
    [Table("work_order_parts")]
    public class WorkOrderPartItem
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("work_order_id")]
        public int WorkOrderId { get; set; }

        [Column("part_id")]
        public int PartId { get; set; }

        [Column("quantity")]
        public int Quantity { get; set; }

        [Column("unit_price")]
        public decimal UnitPrice { get; set; }

        [Column("total_price")]
        public decimal TotalPrice { get; set; }
    }
}

File: DiplomskiAndroid/app/src/main/java/com/example/diplomskiandroid/models/WorkOrderPartItem.java
package com.example.diplomskiandroid.models;

public class WorkOrderPartItem {

    private int id;
    private int workOrderId;
    private int partId;
    private int quantity;
    private double unitPrice;
    private double totalPrice;
    private Part part;

    public int getId() {
        return id;
    }

    public int getWorkOrderId() {
        return workOrderId;
    }

    public int getPartId() {
        return partId;
    }

    public int getQuantity() {
        return quantity;
    }

    public double getUnitPrice() {
        return unitPrice;
    }

    public double getTotalPrice() {
        return totalPrice;
    }

    public Part getPart() {
        return part;
    }
}